spec.html

```diff
@@ -13156,7 +13156,7 @@ <h1>Runtime Semantics: GetTemplateObject ( _templateLiteral_ )</h1>
           1. Let _index_ be 0.
           1. Repeat, while _index_ &lt; _count_,
             1. Let _prop_ be ! ToString(_index_).
-            1. Let _cookedValue_ be the String value _cookedStrings_[_index_].
+            1. Let _cookedValue_ be _cookedStrings_[_index_].
```

Comment:
For posterity, I want to note that this was correct before https://github.com/tc39/ecma262/pull/773; it was an oversight that this was not changed as part of this PR.